include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -185,28 +184,26 @@ inline void UDPSender::send(const std::string& message) noexcept {
         if (m_batchingMessageQueue.empty() || m_batchingMessageQueue.back().length() > m_batchsize) {
             m_batchingMessageQueue.push_back(message);
         } else {
-            std::rbegin(m_batchingMessageQueue)->append("\n").append(message);
+            m_batchingMessageQueue.rbegin()->append("\n").append(message);
         }
     } else {
         sendToDaemon(message);
     }
 }
 
-inline std::optional<std::string> UDPSender::errorMessage() const noexcept {
+inline const std::string& UDPSender::errorMessage() const noexcept {
     return m_errorMessage;
 }
 
 inline bool UDPSender::initialize() noexcept {
-    using namespace std::string_literals;
-
     if (m_isInitialized) {
         return true;
     }
 
     // Connect the socket
     m_socket = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
     if (m_socket == -1) {
-        m_errorMessage = "Could not create socket, err="s + std::strerror(errno);
+        m_errorMessage = std::string("Could not create socket, err=") + std::strerror(errno);
```

Comment:
this was the only place that was making use of `string_literals` in the other cases there was already an `std::string` as one of the operands so `const char*` worked fine